app/src/main/res/layout/fragment_first.xml

```diff
@@ -49,10 +49,19 @@
             android:id="@+id/input"
             android:layout_width="140dp"
             android:layout_height="66dp"
-            android:hint="Team Number"
+            android:hint="teamnumber"
```

Comment:
here it's better for the user to be able to have a clear version, so the old version will be better here